File: campus-second-frontend/src/api/systemintro.ts
import request from '../utils/request'

export interface SystemintroItem {
  id: number
  title: string
  subtitle: string
  content: string
  picture1: string
  picture2: string
  picture3: string
}

export const getSystemintroDetail = () =>
  request.get<any>('/api/systemintro/detail').then((res: any) => res.data ?? res)

export const updateSystemintro = (data: Partial<SystemintroItem>) =>
  request.put<any>('/api/systemintro/update', data).then((res: any) => res.data ?? res)

File: campus-second-backend/src/main/java/com/campus/entity/Systemintro.java
package com.campus.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

/**
 * 系统简介
 */
@Data
@TableName("systemintro")
public class Systemintro {
    @TableId(type = IdType.AUTO)
    private Long id;
    private LocalDateTime addtime;
    private String title;
    private String subtitle;
    private String content;
    private String picture1;
    private String picture2;
    private String picture3;
}
